### PR TITLE
add faq on inc, aop, pan

### PR DESCRIPTION
**Pull request by semaphoreP**
Mostly refer reader to show me the orbit, but also new page on how inc/pan define the orbital plane, and why sine priors on inc.

### CODE REVIEW

**Comment by semaphoreP** [comment]
Hopefully addresses issue #158

**Comment by sblunt** [comment]
Nice, this is awesome. I made two small changes-- 1) removed an unused matplotlib import; 2)slightly changed your explanation of Omega to one that makes more intuitive sense to me. But my explanation is a bit wordy. Let me know what you think; happy to change it back.

**Comment by semaphoreP** [comment]
Looks good to me!
